CSharpSeleniumWebdriverSample/README.md

```diff
@@ -0,0 +1,63 @@
+* [azure-pipelines.yml](./azure-pipelines.yml): Azure Pipelines config file that sets up our Continuous Integration and Pull Request builds
+
+## Tools and libraries used
+
+* [Microsoft.NET.Test.Sdk](https://github.com/microsoft/vstest/) to enable running tests, collect diagnostics data and report results.
+* [Microsoft.NETCore.App](https://dotnet.microsoft.com) a set of .NET API's that are included in the default .NET Core application model.
+* [MSTest.TestAdapter and MSTest.TestFramework](https://github.com/microsoft/testfx) MSTest V2 adapter to discover and execute MSTest Framework based tests.
+* [Selenium.Axe](https://github.com/TroyWalshProf/SeleniumAxeDotnet) to run an axe accessibility scan on the page from the Selenium browser.
+* [Selenium.WebDriver.ChromeDriver](https://github.com/jsakamoto/nupkg-selenium-webdriver-chromedriver/) to enable Selenium to drive Chrome.
+* [Selenium.WebDriver.GeckoDriver](https://github.com/jsakamoto/nupkg-selenium-webdriver-geckodriver/) to enable Selenium to drive Firefox.
+* [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) to run the tests in a CI build with every Pull Request.
+
+## See it in action in Azure Pipelines
+
+[![Build Status](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_apis/build/status/CSharpSeleniumWebdriverSample%20CI?branchName=master)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/latest?definitionId=30&branchName=master)
+
+<!--
+  Note to maintainers: The below example images/links come from a specific build instead of the most recent build so we can link to specific tabs.
+  If you update the links such that they point to a different build, make sure to mark that build as Retained so the links don't expire in a month.
+-->
+The accessibility tests run as part of the `dotnet test` build step:
+
+[![Screenshot of "dotnet test" build logs in sample build](./assets/screenshot-logs-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=2312&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9)
+
+The test pass/fail results display in the Tests tab of the build logs:
+
+[![Screenshot of Tests tab in sample build](./assets/screenshot-tests-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=2312&view=ms.vss-test-web.build-test-results-tab&runId=6462&resultId=100000&paneView=debug)
+
+## See it in action on your local machine
+
+1. Install the stable version of [Chrome](https://www.google.com/chrome/)
+1. Install the stable version of [Firefox] (https://www.mozilla.org/en-US/firefox/)
+1. Clone this sample repository
+
+   ```sh
+   git clone https://github.com/microsoft/axe-pipelines-samples
+   ```
+
+1. Build the Test Project
```

Comment:
`dotnet test` implies the necessary parts of restore and build; you can just say `dotnet test` after the `cd` line, or just run `dotnet test ./csharp-selenium-webdriver-sample/CSharpSeleniumWebdriverSample.csproj` and omit the `cd` as well